docs/user-guide/pods/index.md

```diff
@@ -135,9 +135,9 @@ simplified management.
 
 ## Durability of pods (or lack thereof)
 
-Pods aren't intended to be treated as durable [pets](https://blog.engineyard.com/2014/pets-vs-cattle). They won't survive scheduling failures, node failures, or other evictions, such as due to lack of resources, or in the case of node maintenance.
+Pods aren't intended to be treated as durable [instances](https://blog.engineyard.com/2014/pets-vs-cattle). They won't survive scheduling failures, node failures, or other evictions, such as due to lack of resources, or in the case of node maintenance.
```

Comment:
I agree, the blog link is no longer as relevant, and maybe "entities" rather than "instances" due the the latter being an overloaded term?